templates/server.conf.j2

```diff
@@ -57,3 +59,9 @@ remote-cert-tls client
 management /var/run/openvpn/management unix
 management-client-user root
 {% endif %}
+
+{% if openvpn_use_ldap %}
+### LDAP AUTH ###
+plugin /usr/lib64/openvpn/plugin/lib/openvpn-auth-ldap.so "/etc/openvpn/auth/ldap.conf"
```

Comment:
2 concerns:
1, Does the plugin come included by default in all installs?
2. `lib64` is a bit worrying, not everyone is running this on a 64 bit OS. Could this be retrieved from one of the `ansible_` setup vars?